modules/openid_auth/config/routes.rb

```diff
@@ -7,6 +7,7 @@
     get 'validation', to: 'validation#index'
     get 'mvi-user', to: 'mvi_users#show'
     post 'mvi-user', to: 'mvi_users#search'
+    post 'okta', to: 'okta#okta_callback'
```

Comment:
was able to have a hook hit this route, but still running into the issues mentioned above